internal/pkg/input/validate-aws.go

```diff
@@ -0,0 +1,31 @@
+package pkg
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+)
+
+func ValidateAwsInputCredentials(accessKeyId basetypes.StringValue, secretAccessKey basetypes.StringValue, assumeRoleArn basetypes.StringValue) (AwsAuthenticationMethod, error) {
+
+	// No credentials provided at all
+	if assumeRoleArn.ValueString() == "" && (accessKeyId.ValueString() == "" || secretAccessKey.ValueString() == "") {
+		return "", fmt.Errorf("No credentials provided. Either set access_key_id and secret_access_key, or assume_role_arn.")
+	}
```

Comment:
I think for my case where access_key_id is defined, it should say "Invalid credentials" instead of "No credentials"? 
<img width="841" alt="image" src="https://github.com/user-attachments/assets/2c6ec907-0683-49b5-9249-aeee3629a6fb">